src/util/sortable_collectibles.ts

```diff
@@ -10,20 +10,20 @@ export interface SortableCollectible {
 }
 
 export enum CollectibleSortType {
-    PriceLowToHigh = 'Price: low to high',
-    PriceHighToLow = 'Price: hight to low',
-    NewestAdded = 'Newest Added',
+    PriceLowToHigh = 'price_low_to_high',
+    PriceHighToLow = 'price_high_to_low',
+    NewestAdded = 'newest_added',
 }
 
 export const getCompareFunctionForSort = (sortType: CollectibleSortType) => {
     switch (sortType) {
         case CollectibleSortType.PriceLowToHigh:
-            return (a: SortableCollectible, b: SortableCollectible) => compareAscending(a.price, b.price);
+            return (a: SortableCollectible, b: SortableCollectible) => compareAscending(a.price, b.price, false);
         case CollectibleSortType.PriceHighToLow:
-            return (a: SortableCollectible, b: SortableCollectible) => compareAscending(a.price, b.price) * -1;
```

Comment:
The problem wih this is that the items that had `null` as the `price` (or `creationDate`) appeared at the beginning.